pymc3/distributions/continuous.py

```diff
@@ -4223,6 +4223,28 @@ class Interpolated(BoundedContinuous):
     Both parameters ``x_points`` and values ``pdf_points`` are not variables, but
     plain array-like objects, so they are constant and cannot be sampled.
 
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import pymc3 as pm
+        import arviz as az
+        from scipy.stats import gamma
+        rv = gamma(1.99)
+        x = np.linspace(rv.ppf(0.01),rv.ppf(0.99), 1000)
+        points = np.linspace(x[0], x[-1], 50)
+        pdf = rv.pdf(points)
+        interpolated = pm.Interpolated.dist(points, pdf)
+        plt.style.use('arviz-darkgrid')
+        fig, ax = plt.subplots(1, 1)
+        ax.plot(x, rv.pdf(x), 'C0', linestyle = '--',  label='Original Gamma pdf',alpha=0.8,lw=2)
+        ax.plot(points, pdf, color='black', marker='o', label='Lattice Points',alpha=0.5,linestyle='')
+        ax.plot(x, np.exp(interpolated.logp(x).eval()),'C1',label='Interpolated pdf',alpha=0.8,lw=3)
+        r = interpolated.random(size=1000)
+        ax.hist(r, density=True, histtype='stepfilled', alpha=0.4,align ='right',color='grey')
```

Comment:
I think we want to use `align="mid"` or use the default values for `histtype` and `align`, otherwise the histogram looks shifted. i.e. the leftmost bar of the histogram should coincide with the leftmost bin edge which at the same time should correspond to the leftmost sample in `r`. Same on the right side. I don't think this is currently happening.

You can check by adding `ax.plot(r.min(), 0.2)` and see if it is aligned with the leftmost part of the histogram.

Reply:
yes thankyou, the central align was something i kept from the time I was playing with different distributions and didn't remove it, made the change.